vms/platformvm/health.go

```diff
@@ -5,110 +5,8 @@ package platformvm
 
 import (
 	"context"
-	"errors"
-	"fmt"
-	"strings"
-	"time"
-
-	"go.uber.org/zap"
-
-	"github.com/ava-labs/avalanchego/database"
-	"github.com/ava-labs/avalanchego/utils/constants"
 )
 
-const fallbackMinPercentConnected = 0.8
-
-var errNotEnoughStake = errors.New("not connected to enough stake")
-
-func (vm *VM) HealthCheck(context.Context) (interface{}, error) {
-	// Returns nil if this node is connected to > alpha percent of the Primary Network's stake
-	primaryPercentConnected, err := vm.getPercentConnected(constants.PrimaryNetworkID)
-	if err != nil {
-		return nil, fmt.Errorf("couldn't get percent connected: %w", err)
-	}
-	vm.metrics.SetPercentConnected(primaryPercentConnected)
-	details := map[string]float64{
-		"primary-percentConnected": primaryPercentConnected,
-	}
-
-	localPrimaryValidator, err := vm.state.GetCurrentValidator(
-		constants.PrimaryNetworkID,
-		vm.ctx.NodeID,
-	)
-	switch err {
-	case nil:
-		vm.metrics.SetTimeUntilUnstake(time.Until(localPrimaryValidator.EndTime))
```

Comment:
The reason these metrics were updated here was because this runs periodically (based on time). But I think this was moved to a location that only updates based on transaction issuance... So I feel like this metric would break for local networks (that don't see P-chain traffic)

Reply:
yea true. We use stake metrics in `writeCurrentStakers`, so I thought this could fit there. Reverted those metrics to platform health.